hail/python/hail/docs/change_log.md

```diff
@@ -33,6 +33,27 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+## Version 0.2.119
+
+### New Features
+- (hail#12081) Hail now uses [Zstandard](https://facebook.github.io/zstd/) as
+  the default compression algoritm for table and matrix table storage. Reducing
+  file size around 20% in most cases.
```

Comment:
I think we need a bigger notice of the change of file format. Can we add a new section ### File Format with a message like "The file format is now X.Y.Z. Previous versions of Hail will not be able to read files written by this version of Hail."